### ISSUE TITLE
JS ERROR: TypeError: w.title is null

### ISSUE THREAD

**Issue by jakefoglia** [adrienverge/gnome-magic-window#14]
Thanks for putting together this extension, its great. It certainly has filled the wmctrl hole in my heart. 

I noticed that sometimes my bindings wouldn't work, so I checked out the journalctl logs and saw this: 

`JS ERROR: TypeError: w.title is null`

I inserted some additional logs to see specifically what window(s) have that issue on my system, but I haven't been able to reproduce it yet since restarting my session. I'll keep you updated if I find anything. I imagine the fix might be a simple null check, but I'll do some more digging first.  

Cheers!

System Info:
Arch Linux
Kernel 6.6.17-1-lts
Gnome 45.4 on Wayland
AMD Ryzen 7 PRO 7840U
Radeon 780M Graphics

**Comment by adrienverge** [comment]
Hello Jake and thanks for the report. I don't have a better approach than the one you propose :)
For info, I also have Gnome 45.4 on Wayland, and I don't think I ever encountered this. Maybe it's caused by some specific program that has an untitled window?

**Comment by jakefoglia** [comment]
Hey Adrien! 

So I was able to reproduce the error consistently by opening Mailspring, which was creating a popup dialog box that had a null title attribute. Adding `w.title != null &&` to the filter criteria in `get_windows()` fixed the issue. 

I then realized there is an existing Pull Request for exactly that: https://github.com/adrienverge/gnome-magic-window/pull/9

Would love to see that merged at some point :)
